test/models/vrp_test.rb

```diff
@@ -110,12 +110,12 @@ def test_solver_parameter_retrocompatibility
       vrp = VRP.basic
       generated_vrp = TestHelper.create(vrp)
       assert generated_vrp.resolution_solver
-      assert_nil generated_vrp.preprocessing_first_solution_strategy
+      assert_empty generated_vrp.preprocessing_first_solution_strategy, 'first_solution_strategy'
```

Comment:
In fact this is the improvement :D Minitest already shows the rest of the message automatically thanks to assert_x structure.

When the test breaks instead of just saying 
```
FAIL["test_xxx]
    Expected not nil to be nil
    optimizer-api/test/xxx_test.rb:xxx:in `test_xxx'
```
now it says WHAT was "expected to be empty and not empty" on the output:
```
FAIL["test_xxx]
    first_solution_strategy
    Expected not empty to be empty
    optimizer-api/test/xxx_test.rb:xxx:in `test_xxx'
```

I thought about writing `first_solution_strategy not correct` or `not as expected` but it seemed unnecessary because minitest exactly says that.

Reply:
Oh ok, it is just for us to be able to know that what is not empty, but should be empty, is actually 'first_solution_strategy' ? 
You are right, minitests says the exact same thing. But still I think this can be confusing at first. I let you decide :)

Reply:
do you mean the message or the code is confusing?

Reply:
The message was confusing. But now I get the idea. 
Maybe to avoid any doubt we can re-write `first_solution_strategy not correct` or `not as expected` as you said.

Reply:
I was actually adding a message to make it clear because otherwise we don't know what was expected to be empty without opening the code but I was also trying to minimize the amount of typing because otherwise we don't put any message to our asserts. 

I understand that you have found it confusing at the beginning but now that you got the idea, do you still think it is better to repeat the message? Because one of the utility of assert_x is just that -- i.e., not repeating the 'is not as expected' message for all asserts of type `assert_x` and reporting it automatically in a uniform way.

Reply:
I think it is ok now. If a new developper arrives I think he will easily understand. It is just not intuitive (for me) at the first time but I agree with your reasoning.